ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -485,6 +503,8 @@ public void periodic() {
 
     // refresh the position of the robot
     this.refreshRobotPosition();
+    // update robot position with vision
+    this.addVisionMeasurement(cameraSubsystem.getVisionBotPose());
```

Comment:
Need to protect this line against null reference exception when vision is not in use.

Reply:
already in place with the "useVision" boolean.

Reply:
added a installed hardware check